design/architecture.md

```diff
@@ -669,15 +669,15 @@ a pod is **not** `Privileged` the runtime selection is done as follows:
 
 Kata Containers utilizes the Linux kernel DAX [(Direct Access filesystem)](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/filesystems/dax.txt)
 feature to efficiently map some host-side files into the guest VM space.
-In particular, Kata Containers uses the QEMU nvdimm feature to provide a
+In particular, Kata Containers uses the QEMU NVDIMM feature to provide a
 memory-mapped virtual device that can be used to DAX map the virtual machine's
 root filesystem into the guest memory address space.
 
 Mapping files using DAX provides a number of benefits over more traditional VM
 file and device mapping mechanisms:
 
 - Mapping as a direct access devices allows the guest to directly access
-  the host memory pages (such as via eXicute In Place (XIP)), bypassing the guest
+  the host memory pages (such as via Execute In Place (XIP)), bypassing the guest
```

Comment:
hmm, well the 'X' in eXecute... is for the 'X', not the 'E' ?

Reply:
The original term was misspelt as it should be "eXecute". However, that chokes spell-checkers. The alternatives to what I've written are:

- Put it in backticks (`eXecute`).
- Add "eXecute" to our dictionary.

I don't particularly like either option but if forced, I think I'd have to go with option 2 (even though that seems like overkill for a single instance of a single weirdly written word).

Thoughts?

Reply:
That's just it though - there is only a single instance of this term across all our repos...